src/compiledcode/elf.py

```diff
@@ -715,4 +716,20 @@ class ElfRelocatablesSection(object):
     0804e0b0  00002d07 R_386_JUMP_SLOT   00000000   __strdup
     0804e0b4  00002e07 R_386_JUMP_SLOT   00000000   exit
     """
-    pass
+    def __init__(self):
+        self.readelf_option = '--relocs'
+        self.start_re = re.compile("^Relocation section")
+        self.end_re = EMPTY_LINE_RE()
+        self.sym_names = set()
+        
+    def parse(self, elf, file_like):
+        while 1:
+            line = next_line(file_like)
+            if not line or re.match(self.end_re, line):
+                break
+            line = line.strip()
+            match = RELOC_RE().match(line)
+            if match:
+                sym = match.groups()[4]
```

Comment:
@pombredanne 
Should I distinguished with ''.rel.dyn' and  ''.rel.plt' or more?